components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -1462,193 +1193,198 @@ public void testROIAndRenderingSettingsNoSudo(boolean permWriteOwned, boolean pe
         boolean isExpectSuccessCreateLinkAndChown = isExpectSuccessLinkFileAttachemnt && permChown;
         final EventContext normalUser = newUserAndGroup(groupPermissions);
-        /* set up the light admin's permissions for this test */
+        /* Set up the light admin's permissions for this test.*/
         List<String> permissions = new ArrayList<String>();
         if (permChown) permissions.add(AdminPrivilegeChown.value);
         if (permWriteOwned) permissions.add(AdminPrivilegeWriteOwned.value);
         if (permWriteFile) permissions.add(AdminPrivilegeWriteFile.value);
 
-        /* create an image with pixels as normalUser in a group of the normalUser */
+        /* normalUser creates an image with pixels in normalUser's group.*/
         loginUser(normalUser);
         Image image = mmFactory.createImage();
         Image sentImage = (Image) iUpdate.saveAndReturnObject(image);
-        /* login as light admin */
+        /* Login as lightAdmin.*/
         final EventContext lightAdmin;
         lightAdmin = loginNewAdmin(true, permissions);
         client.getImplicitContext().put("omero.group", Long.toString(normalUser.groupId));
-        /* create a file attachment as light admin */
-        final OriginalFile originalFile = mmFactory.createOriginalFile();
-        FileAnnotation fileAnnotation = new FileAnnotationI();
-        fileAnnotation.setFile(originalFile);
+        /* lightAdmin tries to create a fileAttachment in normalUser's group.*/
+        FileAnnotation fileAnnotation = mmFactory.createFileAnnotation();
+        OriginalFile originalFile = new OriginalFileI();
         try {
             fileAnnotation = (FileAnnotation) iUpdate.saveAndReturnObject(fileAnnotation);
+            originalFile = (OriginalFile) fileAnnotation.getFile();
             Assert.assertTrue(isExpectSuccessCreateFileAttachment);
         } catch (SecurityViolation sv) {
             Assert.assertFalse(isExpectSuccessCreateFileAttachment);
-            return; /* finish the test in case we have no FileAttachment */
+            /* Finish the test in case fileAttachment could not be created.*/
+            return;
         }
-        /* link the file attachment to the image of the user as light admin
+        /* Check that the value of canChown on the fileAnnotation is matching the boolean
+         * isExpectSuccessCreateFileAttAndChown.*/
+        Assert.assertEquals(getCurrentPermissions(fileAnnotation).canChown(), isExpectSuccessCreateFileAttAndChown);
+        /* lightAdmin tries to link the fileAnnotation to the normalUser's image.
          * This will not work in private group. See definition of the boolean
-         * isExpectSuccessLinkFileAttachemnt */
-        ImageAnnotationLink link = new ImageAnnotationLinkI();
-        link.setParent(sentImage);
-        link.setChild(fileAnnotation);
+         * isExpectSuccessLinkFileAttachemnt.*/
```

Comment:
typo ``isExpectSuccessLinkFileAttachemnt`` instead of ``isExpectSuccessLinkFileAttachment``